### ISSUE TITLE
Handling of defines does not accept more than 2 arguments, komodo 2023.04

### ISSUE THREAD

**Issue by mortalisk** [equinor/ert#5253]
**Describe the bug**
If a define has more than 2 values after the key, they should be concatenated. This is not happending in the handle_includes method. Seems to not be a problem on main.

```
ert/gui/main.py", line 155, in _log_difference_with_new_parser
ert_config_new = ErtConfig.from_file(args.config, use_new_parser=True)
File "/prog/res/komodo/2023.04.01-py38-rhel7/root/lib64/python3.8/site-packages/ert/_c_wrappers/enkf/ert_config.py", line 84, in from_file
user_config_dict = ErtConfig.read_user_config(
File "/prog/res/komodo/2023.04.01-py38-rhel7/root/lib64/python3.8/site-packages/ert/_c_wrappers/enkf/ert_config.py", line 245, in read_user_config
return lark_parse(user_config_file, site_config)
File "/prog/res/komodo/2023.04.01-py38-rhel7/root/lib64/python3.8/site-packages/ert/_c_wrappers/enkf/lark_parser.py", line 350, in parse
_handle_includes(tree, pre_defines.copy(), filepath)
File "/prog/res/komodo/2023.04.01-py38-rhel7/root/lib64/python3.8/site-packages/ert/_c_wrappers/enkf/lark_parser.py", line 262, in _handle_includes
args = _substitute_args(args, constraints, defines)
File "/prog/res/komodo/2023.04.01-py38-rhel7/root/lib64/python3.8/site-packages/ert/_c_wrappers/enkf/lark_parser.py", line 238, in _substitute_args
return [
File "/prog/res/komodo/2023.04.01-py38-rhel7/root/lib64/python3.8/site-packages/ert/_c_wrappers/enkf/lark_parser.py", line 239, in <listcomp>
_substitute(defines, x, constraints.expand_envvar)
File "/prog/res/komodo/2023.04.01-py38-rhel7/root/lib64/python3.8/site-packages/ert/_c_wrappers/enkf/lark_parser.py", line 174, in _substitute
for key, val in defines:
ValueError: too many values to unpack (expected 2)
```

**To reproduce**
Steps to reproduce the behaviour:
1. `source /prog/res/komodo/stable/enable`
2. Put `DEFINE <a> b c` in a config
3. `ert gui my_config.ert`

**Expected behaviour**
the define values to be concatenated so the value is "b c"

**Screenshots**

**Comment by DanSava** [comment]
Will this be solved by backporting https://github.com/equinor/ert/pull/5193 ?

**Comment by mortalisk** [comment]
> Will this be solved by backporting #5193 ?

Yes
